data/data/ibmcloud/bootstrap/main.tf

```diff
@@ -67,7 +67,7 @@ resource "ibm_is_security_group" "bootstrap" {
 resource "ibm_is_security_group_rule" "bootstrap_ssh_inbound" {
   group     = ibm_is_security_group.bootstrap.id
   direction = "inbound"
-  remote    = local.public_endpoints ? "0.0.0.0/0" : var.control_plane_security_group_id_list.0.id
+  remote    = local.public_endpoints ? "0.0.0.0/0" : var.control_plane_security_group_id_list[0]
```

Comment:
I don't see any other instance of this variable changed. Is this missing `id` or did the type change?

Reply:
This is actually a bug fix, for a bug that was never noticed until we started testing `Internal`, which hasn't been supported until this and the previous [CORS-2280 PR for Golang changes](https://github.com/openshift/installer/pull/6255).

For instance, using this PR with a revert of the line above, Terraform fails since the `control_plane_security_group_id_list` is a list of Id's (strings) already, not SecurityGroup data sources.
```
# go/src/github.com/openshift/installer/bin/openshift-install version
go/src/github.com/openshift/installer/bin/openshift-install unreleased-master-6395-g75c6ab06c034cfc841b672b936a6a750930606d7
built from commit 75c6ab06c034cfc841b672b936a6a750930606d7
release image registry.ci.openshift.org/origin/release:4.12
release architecture amd64

# git -C go/src/github.com/openshift/installer/ diff cjschaef/ibmcloud_private_terraform..75c6ab06c034cfc841b672b936a6a750930606d7
diff --git a/data/data/ibmcloud/bootstrap/main.tf b/data/data/ibmcloud/bootstrap/main.tf
index 4471b3325..2f0aaea6a 100644
--- a/data/data/ibmcloud/bootstrap/main.tf
+++ b/data/data/ibmcloud/bootstrap/main.tf
@@ -67,7 +67,7 @@ resource "ibm_is_security_group" "bootstrap" {
 resource "ibm_is_security_group_rule" "bootstrap_ssh_inbound" {
   group     = ibm_is_security_group.bootstrap.id
   direction = "inbound"
-  remote    = local.public_endpoints ? "0.0.0.0/0" : var.control_plane_security_group_id_list[0]
+  remote    = local.public_endpoints ? "0.0.0.0/0" : var.control_plane_security_group_id_list.0.id
   tcp {
     port_min = 22
     port_max = 22


# go/src/github.com/openshift/installer/bin/openshift-install create cluster --dir clusters/private-sg-id-list-1/
WARNING Found override for release image. Please be warned, this is not advised 
INFO Consuming Worker Machines from target directory 
INFO Consuming OpenShift Install (Manifests) from target directory 
INFO Consuming Master Machines from target directory 
INFO Consuming Common Manifests from target directory 
INFO Consuming Openshift Manifests from target directory 
INFO Obtaining RHCOS image file from 'https://rhcos.mirror.openshift.com/art/storage/releases/rhcos-4.12/412.86.202208101039-0/x86_64/rhcos-412.86.202208101039-0-ibmcloud.x86_64.qcow2.gz?sha256=09b599849b945bdd405b18765225160f50e07ca205fe9787f70f188c8a96f293' 
INFO The file was found in cache: /root/.cache/openshift-installer/image_cache/rhcos-412.86.202208101039-0-ibmcloud.x86_64.qcow2. Reusing... 
INFO Creating infrastructure resources...         
ERROR                                              
ERROR Error: Unsupported attribute                 
ERROR                                              
ERROR   on main.tf line 70, in resource "ibm_is_security_group_rule" "bootstrap_ssh_inbound": 
ERROR   70:   remote    = local.public_endpoints ? "0.0.0.0/0" : var.control_plane_security_group_id_list.0.id 
ERROR     ├────────────────                        
ERROR     │ var.control_plane_security_group_id_list[0] is "r018-a45f4ff4-b452-44a7-bc12-72bc0e424644" 
ERROR                                              
ERROR Can't access attributes on a primitive-typed value (string). 
ERROR failed to fetch Cluster: failed to generate asset "Cluster": failure applying terraform for "bootstrap" stage: failed to create cluster: failed to apply Terraform: exit status 1 
ERROR                                              
ERROR Error: Unsupported attribute                 
ERROR                                              
ERROR   on main.tf line 70, in resource "ibm_is_security_group_rule" "bootstrap_ssh_inbound": 
ERROR   70:   remote    = local.public_endpoints ? "0.0.0.0/0" : var.control_plane_security_group_id_list.0.id 
ERROR     ├────────────────                        
ERROR     │ var.control_plane_security_group_id_list[0] is "r018-a45f4ff4-b452-44a7-bc12-72bc0e424644" 
ERROR                                              
ERROR Can't access attributes on a primitive-typed value (string).
```